src/logic/logic.js

```diff
@@ -0,0 +1,337 @@
+// function space(len) {
+//   if (!len) {
+//     len = 0;
+//   }
+//   let result = '';
+//   for (let i = 0; i < len; i++) {
+//     result += '   ';
+//   }
+//   return result;
+// }
+function mixColors(color1, color2, weight) {
```

Comment:
I'm guessing we got these code snippets from e.g. https://gist.github.com/jedfoster/7939513 — we should add credit or remove this.